index.css

```diff
@@ -222,3 +220,748 @@ section img {
+.sub1 {
+  width: 295px;
+  height: 87px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 280px;
+  height: 40px;
+}
+
+.tags1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 50px;
+  height: 20px;
+  list-style: none;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.tags1 h6 {
+  width: 32px;
+  height: 20px;
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.tags2 {
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 110px;
+  height: 20px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.tags2 h6 {
+  width: 71px;
+  height: 20px;
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.tags3 {
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 61px;
+  height: 20px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.tags3 h6 {
+  width: 37px;
+  height: 20px;
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.action {
+  position: relative;
+  height: 48px;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: #ff6b00;
+}
+
+.but {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  position: absolute;
+  width: 124px;
+  height: 24px;
+  left: calc(50% - 124px / 2);
+  top: calc(50% - 48px / 2);
+  background: #ff6b00;
+}
+
+.see1 {
+  width: 100px;
+  height: 24px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 2px 5px;
+  background: #1c1a19;
+  position: absolute;
+  width: 375px;
+  height: 563px;
+  left: 0;
+  top: 4300px;
+}
+
+.primary {
+  width: 327px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-size: 72px;
+  line-height: 10px;
+  letter-spacing: 0.37px;
+  margin-bottom: 25px;
+  margin-top: 150px;
+  color: #ff6b00;
+}
+
+.description {
+  width: 327px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 161px;
+  height: 48px;
+  background: #ff6b00;
+}
+
```

Comment:
- [ ] As the last code reviewer said: Please add a hover and pressed state on your buttons to match the design template. You can use the :hover and :active pseudo-classes for this.

![image](https://user-images.githubusercontent.com/58642949/192076733-f42f3158-a9a3-4054-868f-72d518920d6e.png)